csrc/multidevice/executor.cpp

```diff
@@ -39,66 +39,13 @@ copyFusionAndChangeOutputs(Fusion* fusion, const std::vector<Val*>& outputs) {
   // Add new outputs
   std::for_each(outputs.begin(), outputs.end(), [&](Val* const& output) {
     fusion_copy->addOutput(original_to_copy_cloner.clone(output));
-    copy_to_original_map[original_to_copy_cloner.clone(output)] = output;
   });
 
-  for (auto tv : ir_utils::filterByType<TensorView>(fusion_copy->vals())) {
-    tv->setMemoryType(MemoryType::Global);
-    for (auto i : c10::irange(tv->domain()->nDims())) {
-      if (!tv->axis(static_cast<int>(i))->isDeviceDim()) {
-        tv->axis(static_cast<int>(i))->parallelize(ParallelType::Serial);
-      }
-    }
-  }
```

Comment:
Why this was necessary before?

Reply:
I think it was just some dirty hack to try to avoid compilation/lowering error when using FusionExecutorCache due to previous scheduling (while keeping multi-device scheduling because it affects the allocation shape). It is a bit similar to https://github.com/NVIDIA/Fuser/blob/597c9b0c67ae09712688f34aa99da2aaf725f60f/test/multidevice.cpp#L98

This is not needed anyway with this cleaner method